charmcraft/commands/pack.py

```diff
@@ -169,8 +155,13 @@ def _pack_charm(self, parsed_args):
 
     def _pack_bundle(self):
         """Pack a bundle."""
+        project = self.config.project
+        config_parts = self.config.parts.copy()
+        charm_part = config_parts.setdefault("bundle", {})
```

Comment:
This seems like an odd choice of variable name here? Should `charm_part` be `bundle_part`?

Reply:
good catch!

Reply:
Oh, oops, copy-and-paste error. This should be `bundle_part`.